docs/products/kafka/howto/approvals.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Manage approvals
+limited: true
+---
+
+The approvals page allows you to manage requests for Aiven for Apache Kafka® resources owned by your group.
+These requests can include claims for existing topics or requests to create new topics.
+
+## Requests flow
+
+Managing requests involves several key steps:
+
+- **Request initiation**: Users submit a request from the Apache Kafka topic catalog to
+  claim ownership of an existing topic or create a topic.
+- **Request visibility**: All members of the topic's owning group can see the request on
+  the approvals page.
+- **Review and approval**: A group member reviews the request, and either approves or
+  declines it.
+- **Completion**: If approved, the requested topic ownership change or topic creation
+  is completed. If declined, the request is not implemented.
+
+## Key elements
+
+The approvals page displays the following key elements:
+
+- **Topic**: Name of the topic
+- **Type**: Claim or create request
+- **Status**: Current status of the request
+  - **Approved**: Request approved and processed
+  - **Declined**: Request declined. The topic is now available for another group to claim
+  - **Deleted**: Request deleted by the requester
+  - **Failed**: Request cannot complete the required action  (for example, due to
+    Apache Kafka cluster issues or connectivity problems)
+  - **Pending**: Request awaiting approval or decline
+  - **Provisioning**: Request in progress. The topic is being created on the Apache Kafka
+    cluster
+- **Requesting group**: Group that made the request
+- **Requester**: User who submitted the request
+- **Requested on**: Date and time the request was made
+- **Filters**: Filter the list by type, status, requester, and group
+- **Search**: Search for specific topics by name
```

Comment:
```suggestion
- **Filters**: Filter the list by type, status, requester, and group.
- **Search**: Search for specific topics by name.
```

Reply:
I would avoid periods in lists. Following Google styleguide